lib/tapioca/dsl/compilers/active_record_columns.rb

```diff
@@ -254,6 +260,7 @@ def add_methods_for_attribute(klass, column_name, attribute_name = column_name,
             "#{attribute_name}_changed?",
             methods_to_add,
             return_type: "T::Boolean",
+            kw_parameters: [["from", setter_type, "T.unsafe(nil)"], ["to", setter_type, "T.unsafe(nil)"]],
```

Comment:
Based on my suggestion above, this would be:
```suggestion
            parameters: [
              create_kw_opt_param("from", type: setter_type, default: "T.unsafe(nil)"),
              create_kw_opt_param("to", type: setter_type, default: "T.unsafe(nil)")
            ],
```